pmlc/conversion/linalg_to_pxa/linalg_to_pxa.cc

```diff
@@ -338,10 +339,13 @@ struct GenericOpConversion : public OpConversionPattern<GenericOp> {
       op->emitError("No linalg.yield in generic op.");
     }
 
-    for (unsigned i = 0; i < op.getNumResults(); ++i) {
-      op.getResult(i).replaceAllUsesWith(forOp.getResult(i));
+    if (op.getNumResults() == forOp.getNumResults()) {
+      rewriter.replaceOp(op, forOp.getResults());
+    } else {
+      // For some original ops such as conv and copy, they don't return
+      // anything. So we erase the original op directly.
```

Comment:
`linalg.conv` and `linalg.copy` cannot be lowered to PXA because they are already in 'bufferized' form. In other words, we can only lower `linalg` ops that have value-like semantics.